docs/docs/developers/guides/js_apps/use_fee_juice.md

```diff
@@ -71,20 +115,28 @@ import { CLI_Fees } from '/components/snippets';
 
 <CLI_Fees />
```

Comment:
Do you think it makes sense to link to the CLI docs here? So if the flag or anything changes, this wont go stale and we only need to update in 1 place?